docs/hopsworksai/api_key.md

```diff
@@ -0,0 +1,62 @@
+# Hopsworks.ai API Key
+
+Hopsworks.ai allows users to generate an API Key that can be used to authenticate and access the Hopsworks.ai REST APIs.
+
+## Generate an API Key
+
+This section describes how to generate an API Key for Hopsworks.ai. First, login to your Hopsworks.ai account, then click on the Settings tab as shown below:
```

Comment:
I would put the first sentence on its own line.
```suggestion
This section describes how to generate an API Key for Hopsworks.ai. 

First, login to your Hopsworks.ai account, then click on the Settings tab as shown below:
```

Reply:
Given the shortness of the section and the title I think you could also skip the first sentence.